pysatNASA/instruments/icon_mighti.py

```diff
@@ -220,106 +310,44 @@ def load(fnames, tag=None, sat_id=None, keep_original_names=False):
         inst.load(2020, 1)
 
     """
+    labels = {'units': ('Units', str), 'name': ('Long_Name', str),
+              'notes': ('Var_Notes', str), 'desc': ('CatDesc', str),
+              'min_val': ('ValidMin', float),
+              'max_val': ('ValidMax', float), 'fill_val': ('FillVal', float)}
+
+    data, meta = pysat.utils.load_netcdf4(fnames, epoch_name='Epoch',
+                                          pandas_format=pandas_format,
+                                          labels=labels)
 
-    data, mdata = pysat.utils.load_netcdf4(fnames, epoch_name='Epoch',
-                                           units_label='Units',
-                                           name_label='Long_Name',
-                                           notes_label='Var_Notes',
-                                           desc_label='CatDesc',
-                                           plot_label='FieldNam',
-                                           axis_label='LablAxis',
-                                           scale_label='ScaleTyp',
-                                           min_label='ValidMin',
-                                           max_label='ValidMax',
-                                           fill_label='FillVal',
-                                           pandas_format=pandas_format)
     # xarray can't merge if variable and dim names are the same
     if 'Altitude' in data.dims:
         data = data.rename_dims(dims_dict={'Altitude': 'Alt'})
-    return data, mdata
 
+    return data, meta
 
-def clean(inst):
-    """Provides data cleaning based upon clean_level.
 
-    Routine is called by pysat, and not by the end user directly.
+# ----------------------------------------------------------------------------
+# Local functions
+
+
+def remove_preamble(inst):
+    """Removes preambles in variable names
```

Comment:
This could also live in icon methods.

Reply:
I've moved the routines for FUV and MIGHTI to a single routine under methods.  IVM and EUV don't have this function